ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywall.kt

```diff
@@ -69,7 +68,6 @@ private fun LoadedPaywall(state: PaywallState.Loaded, viewModel: PaywallViewMode
                         ),
                     )
                     .background(backgroundColor)
-                    .padding(top = UIConstant.footerRoundedBorderHeight)
```

Comment:
We can't add a padding here because it wouldn't apply the clip to the entire thing.

Reply:
Hmm interesting, I'm not sure when that broke, but it did apply the clip to the padding before :man_shrugging: